src/pages/MainPage/MainPage.js

```diff
@@ -1,9 +1,240 @@
+                </button>
+              </li>
+            </ol>
+          </div>
+          <ul className="newProducts" style={saleStyle}>
+            {saleList.map(saleItem => (
+              <li key={`${saleItem.id}`}>
+                <Link to="#">
+                  <figure className="newProductConent">
+                    <img src={saleItem.url} alt={saleItem.title} />
+                    <figcaption>
+                      <strong>{saleItem.title}</strong>
+                    </figcaption>
+                  </figure>
+                </Link>
+              </li>
+            ))}
+          </ul>
+        </section>
+        <section className="newCollectionWrapper">
+          <div className="newCollectionBg" />
+          <div className="newCollectionContent">
+            <span>LAST CHANCE</span>
+            <h2>SS21 시즌오프 세일 연장</h2>
+            <span>마지막 시즌 오프 세일 찬스 놓치지 마세요!</span>
+            <div className="btnGroup">
+              <button type="button" className="saleManBtn">
+                남성 세일
+              </button>
+              <button type="button" className="saleWomenBtn">
+                여성 세일
+              </button>
+            </div>
+          </div>
+        </section>
+        <section className="lafestWorldWrapper">
+          <h2>The Lacoste World</h2>
+          <ul className="lafeWorlds">
+            {worldList.map(item => (
+              <li key={item.id}>
+                <Link to="#">
+                  <figure>
+                    <img src={item.url} alt={item.title} />
+                    <figcaption className="lafestWorlContent">
+                      <h3>{item.title}</h3>
+                      <strong>신상품 7% 혜택</strong>
+                      <button type="button">구매하기</button>
+                    </figcaption>
+                  </figure>
+                </Link>
+              </li>
+            ))}
+          </ul>
+        </section>
+        <section className="lafestInsideWrapper">
+          <div>
+            <h3>Lacoste Inside</h3>
+            <ol className="indications">
+              <li>
+                <button
+                  type="button"
+                  className="prevBtn"
+                  onClick={this.handleInSidePrevClick}
+                  disabled={insideCount === 1 && true}
+                >
+                  <i className="fas fa-chevron-left" />
+                </button>
+              </li>
+              <li className="startCount">
+                <span>{insideCount}</span>
+              </li>
+              <li className="center">
+                <span>/</span>
+              </li>
+              <li className="endCount">
+                <span>{insideList.length}</span>
+              </li>
+              <li>
+                <button
+                  type="button"
+                  className="nextBtn"
+                  onClick={this.handleInSideNextClick}
+                  disabled={insideCount === insideList.length && true}
```

Comment:
```suggestion
                  disabled={insideCount === insideList.length}
```
연산 결과로 떨어진 boolean 값을 그대로 이용해도 되겠네요